src/items.h

```diff
@@ -283,6 +283,10 @@ class ItemType
 				return name;
 			}
 
+			if (name.empty() || name.back() == 's') {
```

Comment:
Wouldn't it make more sense if the console would be showing information that an item has no name set but a plural name set, other than that it wouldn't make much sense to check for ```name.empty()``` or am I misstaken?

Reply:
You can't call name.back() on an empty string, so empty string case must be handled and I think returning an empty string for plural name makes more sense than returning "s" which is the current behavior.

Reply:
I'm not opposed to logging if there is an item with an empty name though, not sure how common that is and if it would be of use.